frontend/components/events/EventDetails.tsx

```diff
@@ -16,7 +16,7 @@ export function EventDetails(props: EventDetailsProps) {
   const { eventData, eventTags, setLoading } = props;
   return (
     <div className="flex justify-center w-full">
-      <div className="pb-10 screen-width-primary">
+      <div className="pb-10 screen-width-primary sm:rounded-3xl">
```

Comment:
whats this for?

Reply:
it was there before but i removed it so now im putting it back

Reply:
isnt it in `EventImage`? i see u removed it there..?